apps/vor/pkg/postgres/schoolStore.go

```diff
@@ -7,11 +7,23 @@ import (
 	"time"
 )
 
-type SchoolStore struct {
-	*pg.DB
-}
+type (
+	SchoolStore struct {
+		*pg.DB
+	}
 
-func (s SchoolStore) NewSchool(schoolName string, userId string) (*School, error) {
+	GuardianRelation struct {
```

Comment:
I think `GuardianWithRelation` might be a more appropriate name for this. Since it includes guardian data and their relationship with a student.